Added asc and desc option in AssignmentOverview and Assignment

Larspolo opened the pull request:
## Description
Added a order type button and moved some buttons around to make it look all nice and pretty again :wink: 
![image](https://user-images.githubusercontent.com/11822478/45226339-df055880-b2be-11e8-810f-9487a54e86d5.png)


## Summary of changes
- Fixes #183

Airgetfrog commented on src/vue/src/views/Assignment.vue:
```diff
@@ -4,20 +4,30 @@
+                    <input class="theme-input full-width multi-form" type="text" v-model="searchVariable" placeholder="Search..."/>
+                </b-col>
+                <b-col sm="8">
+                    <b-form-select class="multi-form btn" v-model="selectedSortOption" :select-size="1">
```
These probably shouldn't be classed as `btn`'s, it causes the entire dropdown to have a blue background while active. Ditto for the one below

engelhamer replied:
I guess we should copy the required styling from the btn class (in buttons.sass) to the select tag (in select.sass) and indeed leave the changing background behind.

Airgetfrog commented on src/vue/src/views/AssignmentsOverview.vue:
```diff
@@ -4,16 +4,27 @@
+                        <input class="theme-input full-width multi-form" type="text" v-model="searchVariable" placeholder="Search..."/>
+                    </b-col>
+                    <b-col sm="8">
+                        <b-form-select class="btn" v-model="selectedSortOption" :select-size="1">
```
Same as above

Airgetfrog commented:
Probably good to add this to "manage course members" too: 

![image](https://user-images.githubusercontent.com/29678823/45230245-918de900-b2c8-11e8-8295-d03a6fccaaa0.png)

Also, I'd prefer using a simple/more generic "ascending/descending" over the current "A-Z/1-9".

Larspolo commented:
Fixed manage course members and implemented the other requested changes
![image](https://user-images.githubusercontent.com/11822478/45237029-e0924900-b2dd-11e8-8307-707217000ed7.png)

Larspolo commented:
![image](https://user-images.githubusercontent.com/11822478/45241593-46d29800-b2ed-11e8-96a0-941f590647b5.png)
![image](https://user-images.githubusercontent.com/11822478/45241600-4b974c00-b2ed-11e8-8ec0-77b512d3a5b5.png)
Added icons
